src/cli/build.ts

```diff
@@ -111,7 +103,9 @@ export default function (program: RootCmd) {
         // only actually build/push if there are files to include
         if (includedFiles.length > 0) {
           await createDockerfile(cfg.pepr.peprVersion, cfg.description, includedFiles);
-          execSync(`docker build --tag ${image} -f Dockerfile.controller .`, { stdio: "inherit" });
+          execSync(`docker build --tag ${image} -f Dockerfile.controller .`, {
```

Comment:
What happened to the customImage flag?

Reply:
I removed it by accident when I was fixes merge conflicts yesterday.